design-system/src/pattern-library/foundations/buttons.html

```diff
@@ -47,7 +47,7 @@ <h2 class="Pen-panel-title pull-left">Example</h2>
                             <h3 class="tabs-container__title" id="TabPanelTitle-Tab1">Example</h3>
                             <div class="tabs-container__panel" id="Tab1">
                                 <div class="coop-l-grid">
-                                    <div class="coop-l-grid__item coop-l-grid__item--medium-8 coop-l-grid__item--large-6">
+                                    <div class="coop-l-grid__item coop-l-grid__item--medium-12 coop-l-grid__item--large-8">
```

Comment:
Any grid item spans 12 columns by default from the lowest screen size, so this can be simplified to:
```suggestion
                                    <div class="coop-l-grid__item coop-l-grid__item--large-8">
```